splice_spark/src/main/spark2.3/com/splicemachine/spark/splicemachine/SplicemachineContext.scala

```diff
@@ -356,28 +365,28 @@ class SplicemachineContext(options: Map[String, String]) extends Serializable {
     path.toString
   }
 
-  def getRandomName(): String = {
+  private[this] def getRandomName(): String = {
     val name = new Array[Byte](32)
     new SecureRandom().nextBytes(name)
     Bytes.toHex(name)+"-"+System.nanoTime()
   }
 
   /**
-    * SQL to Dataframe translation.  (Lazy)
+    * SQL to Dataset translation.  (Lazy)
     *
-    * @param sql
-    * @return
+    * @param sql SQL query
+    * @return Dataset[Row] with the result of the query
```

Comment:
`Dataset[Row]` is included in the signature so there's no need to include it in the `@return`. Moreover `Dataset[Row]` does not include anything but describes a distributed computation that once executed (e.g. `show`) gives a result (aka _query result_). I'd simply remove the line as it adds nothing at all.